backend/apps/search/migrations/0004_searchdocument_add_description_tags_typename.py

```
@@ -0,0 +1,31 @@
+# Generated manually — adds description, tags, and content_type_name to SearchDocument
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("search", "0003_searchanalytics"),
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)